airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -111,6 +111,12 @@
         "examples": ["10"],
         "description": "Optional. Increase this if syncing tables larger than 100GB. Only relevant for COPY. Files are streamed to S3 in parts. This determines the size of each part, in MBs. As S3 has a limit of 10,000 parts per file, part size affects the table size. This is 10MB by default, resulting in a default limit of 100GB tables. Note, a larger part size will result in larger memory requirements. A rule of thumb is to multiply the part size by 10 to get the memory requirement. Modify this with care.",
         "title": "Stream Part Size"
+      },
+      "purge_staging_data": {
+        "title": "Purge Staging Files and Tables",
+        "type": "boolean",
+        "description": "Only relevant for COPY. Defaults to true.",
```

Comment:
we should say more here to make impact clearer e.g: "Whether to delete the staging files written to S3 after each sync. See <the docs> for more information" and also there should be a docs section explaining the user impact e.g:  what output format should they expect to find ?